homeassistant/components/solaredge/const.py

```diff
@@ -51,7 +51,7 @@
         json_key="lastYearData",
         name="Energy this year",
         entity_registry_enabled_default=False,
-        icon="mdi:solar-power",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
```

Comment:
Lifetime energy already provides this state class, why add it here? As in, this is not needed to add to the dashboard.

Additionally, this sensor is not solely increasing, it can go down because of the power consumption of the inverter (e.g., during the night). Therefore, this state class should not be used.